mono/metadata/threads.c

```diff
@@ -95,7 +95,8 @@ extern int tkill (pid_t tid, int signal);
 
 #define SPIN_UNLOCK(i) i = 0
 
-#define LOCK_THREAD(thread) lock_thread((thread))
+#define TRY_LOCK_THREAD(thread) try_lock_thread((thread))
```

Comment:
Macros need parens on the outside, but the parameter only needs one set, not two

Reply:
I'll fix the UNLOCK_THREAD beneath this as well then